src/dotnet-interactive/Program.cs

```diff
@@ -30,6 +31,16 @@ public class Program
     public static async Task<int> Main(string[] args)
     {
         Console.OutputEncoding = Encoding.UTF8;
+
+        var culture = Environment.GetEnvironmentVariable("DOTNET_CLI_CULTURE");
+        var uiLanguage = Environment.GetEnvironmentVariable("DOTNET_CLI_UI_LANGUAGE");
+
+        if (!string.IsNullOrWhiteSpace(culture) && !string.IsNullOrWhiteSpace(uiLanguage))
```

Comment:
These can be set independently so splitting this into separate `if` blocks would be a good idea.

Reply:
changed